File: gaia-sdk-python/graphql/response/type/SkillEvaluation.py

from graphql.response.type.SyncSkillEvaluation import SyncSkillEvaluation
from graphql.response.type.AsyncSkillEvaluation import AsyncSkillEvaluation


class SkillEvaluation:
    def sync(self) -> SyncSkillEvaluation:
        return self.sync
    def async(self) -> AsyncSkillEvaluation:
        return self.async

File: gaia-sdk-java/gaia-sdk-graphql/src/main/kotlin/gaia/sdk/response/type/SkillEvaluation.kt
package gaia.sdk.response.type

import gaia.sdk.Uuid
import gaia.sdk.Timestamp
import gaia.sdk.Long
import gaia.sdk.Struct
import gaia.sdk.response.intf.*
import com.fasterxml.jackson.annotation.JsonProperty
import com.fasterxml.jackson.annotation.JsonCreator
import gaia.sdk.request.enumeration.*

data class SkillEvaluation @JsonCreator constructor(
    @JsonProperty("syncEval") val syncEval:SyncSkillEvaluation?, 
    @JsonProperty("asyncEval") val asyncEval:AsyncSkillEvaluation?
)